config/schedule.rb

```diff
@@ -35,6 +35,18 @@ def local(time)
   runner "PatientDeduplication::Runner.new(PatientDeduplication::Strategies.identifier_and_full_name_match).perform"
 end
 
+every :day, at: local("02:30 am").utc, roles: [:cron] do
+  if Flipper.enabled?(:live_experiment)
+    runner "ExperimentControlService.start_medication_reminder_experiment('name')" # needs the real experiment name
```

Comment:
What is the significance of the name? If its just arbitrary, I would make it a constant and encapsulate it inside the experiment object.  None of the clients need to know about it or care.

```ruby
  ExperimentControlService.start_medication_reminder_experiment
...
class Experiment
  MEDICATION_REMINDER = "medication_reminder"
 
  def self.medication_reminder_experiment
    create_or_find_by(name: MEDICATION_REMINDER)
  end
```

Reply:
I didn't do this because the experiment needs to be setup manually anyhow. Instead, I just got rid of the name argument and I'm just looking up the first medication reminder experiment. If we end up creating more than one medication reminder experiment, we can address the issue at that time.